interfaces/repo.go

```diff
@@ -363,17 +359,14 @@ func (r *Repository) AddSlot(slot *Slot) error {
 	if i == nil {
 		return fmt.Errorf("cannot add slot, interface %q is not known", slot.Interface)
 	}
-	if err := slot.Sanitize(i); err != nil {
```

Comment:
Why are we removing this?

Reply:
This is now done in info.ReadInfo (per our plan from the forum).